tests/integration/records/table_validator.py

```diff
@@ -1,20 +1,55 @@
 import pytz
 import datetime
 import logging
+from sqlalchemy.engine import Engine
```

Comment:
This file, one of the 3 largest in the project, increased in size to 366 lines.  The total size of those files is now 3408 lines (target: 875).  Is this file complex enough to refactor?